### ISSUE TITLE
properties not allowed when Inventory controller

### ISSUE THREAD

**Issue by robinlem** [glpi-project/glpi-agent#145]
### Bug reporting acknowledgment

Yes, I read it

### Describe the bug

Hi,

When sending an inventory. This is the case on my physical DELL PowerEdge machine. I get the following error.

ERROR: JSON does not validate. Violations:
Additional properties not allowed: model at #->properties:content->properties:controllers->items[0]:0

This is an extract of the firsts controllers in local inventory. 

```
      "controllers": [
         {
            "manufacturer": "Dell",
            "model": "04KF8J",
            "name": "QLogic 2x25GE QL41262HMCU CNA",
            "rev": "A02",
            "serial": "MY135549AJ9NTM",
            "type": "NDC"
         },
         {
            "manufacturer": "Dell",
            "model": "0K2Y8N",
            "name": "DRIVE BACKPLANE",
            "rev": "A00",
            "serial": "CNIVC009AA0394",
            "type": "BP"
         },
         {
            "caption": "Sky Lake-E DMI3 Registers",
            "manufacturer": "Intel Corporation",
            "name": "Sky Lake-E DMI3 Registers",
            "pciclass": "0600",
            "pcislot": "00:00.0",
            "pcisubsystemid": "8086:0000",
            "productid": "2020",
            "rev": "07",
            "type": "Host bridge",
            "vendorid": "8086"
         },
         {
            "caption": "Sky Lake-E MM/Vt-d Configuration Registers",
            "manufacturer": "Intel Corporation",
            "name": "Sky Lake-E MM/Vt-d Configuration Registers",
            "pciclass": "0880",
            "pcislot": "00:05.0",
            "pcisubsystemid": "8086:0000",
            "productid": "2024",
            "rev": "07",
            "type": "System peripheral",
            "vendorid": "8086"
         },
         {

```

Has anyone encountered this bug before?

Regards

Robin

### To reproduce

1 install glpi 10.0.0
2 download inventory agent 1.2
3 install agent on physical machine DELL PowerEdge with ubuntu 20.04
4 send inventory to GLPI

### Expected behavior

an import of the inventory 

### Operating system

Linux

### GLPI Agent version

1.2

### GLPI version

10.0.0

**Comment by g-bougard** [comment]
Hi @robinlem 

this is a server issue and a duplicated of fixed upstream glpi-project/glpi#11811.

You should wait for next GLPI 10.0.2 and upgrade it. It will be available soon.

**Comment by robinlem** [comment]
Great, thx Guillaume
